ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1577,10 +1577,9 @@ def test_tensorflow_realdiv(
     fn_tree,
     on_device,
 ):
-    # todo: test for complex numbers
     input_dtype, x = dtype_and_x
     helpers.test_frontend_function(
-        input_dtypes=input_dtype,
+        input_dtypes=helpers.get_dtypes("float_and_complex"),
```

Comment:
You don't generate the dtype here, you do it in the decorator above in the `available_dtype` arguemtns